debian/control

```diff
@@ -55,13 +55,13 @@ Description: Whonix configuration for SecureDrop.
  
 Package: securedrop-workstation-config
 Architecture: all
-Depends: ${python3:Depends}, python3-qubesdb, rsyslog, mailcap, apparmor, nautilus, securedrop-keyring, xfce4-terminal
+Depends: ${python3:Depends}, python3-qubesdb, rsyslog, mailcap, apparmor, securedrop-keyring, xfce4-terminal
 Description: This is the SecureDrop workstation template configuration package.
  This package provides dependencies and configuration for the Qubes SecureDrop workstation VM Templates.
 
 Package: securedrop-workstation-viewer
 Architecture: all
-Depends: securedrop-workstation-config,securedrop-workstation-grsec,apparmor-profiles,apparmor-profiles-extra,apparmor-utils,audacious,eog,evince,file-roller,gedit,totem
+Depends: securedrop-workstation-config,securedrop-workstation-grsec,apparmor-profiles,apparmor-profiles-extra,apparmor-utils,audacious,eog,evince,file-roller,gedit,totem, thunar
```

Comment:
We should probably settle on GNOME vs XFCE now, because this makes it clear there's a divergence given eog, evince, gedit, etc. are all pulling in the GNOME stack regardless.

My opinion is (still) that GNOME is better for usability.

Reply:
If we can a fix in for nautilus onder the current grsec kernel we can swap it back in instead. I'm not too concerned about which we do go with. My two hills here are (a) it shouldn't be available outside the specific VMs where there's a legit use for it and (b, related) we should continue to install an as minimal as possible set of packages in the base template and its children.

Reply:
Right, that's where the disconnect is. I think usability is a much larger factor for our audience. I also don't follow/agree with the implication that minimal means more secure/better. (And personally having a file manager is on par with having a usable terminal, c.f. 5b85470bf35e9f54a3a6ca9b471bccc0f85d1f66.)

> If we can a fix in for nautilus under the current grsec kernel we can swap it back in instead

OK, here you go: https://github.com/freedomofpress/securedrop-client/pull/2060

Reply:
I'll review #2060 and flip this back to nautilus if all is good. Couple of points tho:
1) we're explicitly removing sd-app (and applications) from the Qubes menu, because the only way users are expected to interact with it is via securedropc-client and the desktop icon. So if we install nautilus in sd-app it won't be available to users without dropping into a terminal anyway.
2) Usability also implies removing incorrect choices - if we *did* have a file manager listed for each sd-* vm, it would be less clear to folks where they should be interacting with files directly. In the context of SDW, the correct places are sd-devices and maybe sd-viewer. As currently designed, there's no scenario in which users should be interacting with the file store on sd-app, except via the client.

It's definitely a pain for development and also for users who are comfortable with doing their own thing in Qubes, but IMO that pain is worth it to make the happy paths more obvious and safer for general users.